subjects/PE-x86/VCExeSample/VCExeSample.c

```diff
@@ -10,9 +10,9 @@ int32 main(int32 argc, char * * argv)
 	return 0x00;
 }
 
-void test1(char * arg1, int32 arg2, char * arg3, real32 arg4)
+void test1(char * arg1, int32 arg2, char * arg3, Eq_10 arg4)
 {
-	printf("%s %d %s %f");
+	printf("%s %d %s %f", arg1, arg2, arg3, arg4);
 	return;
```

Comment:
The varargs is not specific case for it. The same problem could be reproduced on any procedures with double formal arguments and float actuals